sdk/src/wallet/core/mod.rs

```diff
@@ -360,37 +344,12 @@ where
     crate::wallet::Error: From<S::Error>,
     crate::client::Error: From<S::Error>,
 {
-    /// Create a new wallet.
-    pub(crate) async fn new(inner: Arc<WalletInner<S>>, data: WalletData) -> Result<Self> {
```

Comment:
Why wasn't it used anymore btw?

Reply:
https://github.com/iotaledger/iota-sdk/issues/2022